### ISSUE TITLE
Misc ideas based on discussions in Slack and TODOs

### ISSUE THREAD

**Issue by osanseviero** [huggingface/community-events#4]
Just dumping ideas here related to this repo cc @NielsRogge @nateraw 

- [ ] Add link to projects spreadsheet
- [ ] Add important dates at https://github.com/huggingface/community-events/tree/main/huggan#important-dates
- [ ] I would suggest splitting keras and pytorch uploading into a 2.1 and 2.2 sections for clearer separation, and have model card in a third section
- [ ] Have a third uploading section showing how to do just with git add, git commit, etc. Maybe we can link to the "Add a model" guide. The `PyTorchModelHubMixin` has not been widely tested so I'm a bit worried people face issues.
- [ ] Add a template model card
- [ ] Determine which tags we want for all repos to have
- [ ] Clarify between the two tracks (or decide if we just want one track)
- [ ] Add info on prizes
- [ ] Add info on evaluation
- [ ] Add link to talks
- [ ] Add link to Discord channel
- [ ] Add some instructions (or video) on how to use the GPUs (inspired in https://github.com/huggingface/transformers/tree/main/examples/research_projects/robust-speech-event)
- [ ] Add some section on "Communication and Problems" similar to https://github.com/huggingface/transformers/tree/main/examples/research_projects/robust-speech-event#communication-and-problems
- [ ] Add clarifications of what GANs users can train
- [ ] Add some table linking to the example training scripts

**Comment by NielsRogge** [comment]
Adding some more ideas:

- [ ] Make people learn about GAN metrics, like FID and Inception score
- [ ] Add section related to the main applications of GANs
- [ ] Explain how to set up Accelerate to set up the scripts

**Comment by osanseviero** [comment]
Some new todos (lol the list grows larger and larger) based on internal discussion https://huggingface.slack.com/archives/C035Y01JHMY/p1648122316988469

- [ ] Allow uploading multiple TF models to a single model repo (solved with https://github.com/huggingface/huggingface_hub/issues/533) or just the generator
- [ ] validate scripts work on multiple GPUs
- [ ] add automatic model card in scripts

**Comment by lhoestq** [comment]
> Tag all vision datasets on the hub with "vision" such that people can easily retrieve them

Currently we don't tag datasets with a "image" or "text" or "audio" tag but we can probably do that indeed. Maybe
```yaml
modalities:
- image
```
for example ? Or you have something else in mind ?

**Comment by NielsRogge** [comment]
> for example ? Or you have something else in mind ?

Maybe "vision"? Currently, all vision models on the hub are tagged with that.

**Comment by lhoestq** [comment]
Ok sounds good :)
I'll also adapt the [datasets tagging app](https://huggingface.co/spaces/huggingface/datasets-tagging) to support this field then, and make it compatible for vision datasets (right now it's heavily focused on text datasets)

**Comment by julien-c** [comment]
(if we move to the new task scheme that you proposed recently, we don't need to hardcode the modality b/c it will be implied by the task)

**Comment by julien-c** [comment]
(cc @osanseviero )

**Comment by lhoestq** [comment]
> (if we move to the new task scheme that you proposed recently, we don't need to hardcode the modality b/c it will be implied by the task)

Yes indeed, maybe we don't even need the modality tag if we can just infer it from the task tag, as we do for models ^^
